src/modules/laserHokuyo/laserHokuyo.cpp

```diff
@@ -42,8 +42,15 @@ bool laserHokuyo::open(yarp::os::Searchable& config)
     //list of mandatory options
     //TODO change comments
     period = general_config.check("Period", Value(50), "Period of the sampling thread").asInt();
+
+    if (general_config.check("max_angle") == false) { yError() << "Missing max_angle param"; return false; }
+    if (general_config.check("min_angle") == false) { yError() << "Missing min_angle param"; return false; }
+    max_angle = general_config.find("max_angle").asDouble();
+    min_angle = general_config.find("min_angle").asDouble();
+
```

Comment:
Technically, this change adds parameters to the device, therefore it does not belong to `master`...